config.go

```diff
@@ -43,7 +43,7 @@ const (
 	defaultBlockMaxSize      = 375000
 	blockMaxSizeMin          = 1000
 	blockMaxSizeMax          = wire.MaxBlockPayload - 1000
-	defaultBlockPrioritySize = 50000
+	defaultBlockPrioritySize = 0
```

Comment:
I disagree. Both stake and regular transactions have an associated priority, and in this case there are two new functions that always consider the stake importance and then consider the respective priority or fees per KB, depending on whether or not the user elects to have a priority region in the block. Based on feedback from miners, they all would rather sort by fees *first* and then by priority.